### ISSUE TITLE
How to open tab through href

### ISSUE THREAD

**Issue by theallseingeye** [reactstrap/reactstrap#1399]
I'm using NavTabs to toggle <Tabs>

Is there a way I can directly open the Tab by using /link#mytabid and when I select a tab is it possible to scroll to the beginning of the content?

**Comment by TheSharpieOne** [comment]
Yes, you control which tab is active. When the page loads (and/or when the hash changes) you can grab the value from `window.location.hash` to determine which tab should be active.
You also control when tabs a switched (by changing the value of the active tab). When you do this, you can also scroll the window via `element.scrollIntoView()` or some other means (`window.scrollTo(x, y)`)

**Comment by theallseingeye** [comment]
Do you have any example?
